game/beeb_core.asm

```diff
@@ -893,4 +893,73 @@ ENDIF
     RTS  
 }
 
+
+; X=column [0-79]
+; A=offset [0-1]
+.beeb_dhires_copy_column
+{
+    CLC
+    ADC Mult8_LO, X
+    STA beeb_writeptr
+    LDA #HI(beeb_double_hires_addr)
+    ADC Mult8_HI, X
+    STA beeb_writeptr+1
+
+    LDX #BEEB_DOUBLE_HIRES_ROWS
+    .row_loop
+
+    LDY #0
+    .char_loop
+    LDA (beeb_writeptr), Y
+    PHA
+
+    \\ Write to visible screen
+    LDA &FE34:EOR #&4:STA &FE34
+
+    PLA
+    STA (beeb_writeptr), Y
+
+    \\ Read from invisible screen
+    LDA &FE34:EOR #&4:STA &FE34
+
+    INY:INY
+    CPY #8
+    BCC char_loop
+
+    CLC
```

Comment:
:exclamation: Total flyby microoptimization alert :exclamation: 

You can abuse (I think) the fact that carry's definitely set here, and save the clear carry (and doing `ADC #LO(BEEB_SCREEN_ROW_BYTES)-1`) I think :grinning: ...

A whole byte!!!11!1